package.json

```diff
@@ -130,5 +130,15 @@
     "vinyl-buffer": "^1.0.1",
     "vinyl-source-stream": "^2.0.0",
     "yargs": "^16.0.3"
-  }
+  },
+  "files": [
+    "/dist/fonts",
+    "/dist/img",
+    "/dist/js",
+    "/dist/scss",
+    "/src/js",
```

Comment:
Shouldn't we modify the build task to throw `src/js/components` into `dist` so it ends up being:

```
.
├── js
├── uswds.js
├── uswds.min.js
├── uswds.min.js.map
└── components/
    ├── accordion.js
    ├── banner.js
    ├── charaacter-count.js
    └── …
```